components/dirigeants-section/elus-section.tsx

```diff
@@ -65,11 +64,38 @@ const ElusSection: React.FC<{ uniteLegale: IUniteLegale }> = ({
         )}
       </Section>
       <style global jsx>{`
-        table > tbody > tr > td:first-of-type {
-          width: 30%;
+        table > tbody > tr > td {
+          min-width: 30%;
         }
       `}</style>
```

Comment:
Hum this is dangerous as there is a `global` keyword so the style will apply to every `table > tbody > tr > td`

This was poorly placed in the first place (my fault!) but you should check if this does not break table with multiple columns elsewhere